From: Martin Botka <martin.botka@somainline.org>
To: Mark Rutland <mark.rutland@arm.com>,
	 Lorenzo Pieralisi <lpieralisi@kernel.org>,
	 Sudeep Holla <sudeep.holla@arm.com>,
	 "Rafael J. Wysocki" <rafael@kernel.org>,
	 Viresh Kumar <viresh.kumar@linaro.org>,
	Yangtao Li <tiny.windzz@gmail.com>,  Chen-Yu Tsai <wens@csie.org>,
	Jernej Skrabec <jernej.skrabec@gmail.com>,
	 Samuel Holland <samuel@sholland.org>,
	Rob Herring <robh+dt@kernel.org>,
	 Krzysztof Kozlowski <krzysztof.kozlowski+dt@linaro.org>,
	 Conor Dooley <conor+dt@kernel.org>
Cc: linux-arm-kernel@lists.infradead.org,
	linux-kernel@vger.kernel.org,  linux-pm@vger.kernel.org,
	linux-sunxi@lists.linux.dev,  devicetree@vger.kernel.org,
	Andre Przywara <andre.przywara@arm.com>,
	 Alan Ma <tech@biqu3d.com>,
	Luke Harrison <bttuniversity@biqu3d.com>,
	 Marijn Suijten <marijn.suijten@somainline.org>,
	 AngeloGioacchino Del Regno
	<angelogioacchino.delregno@somainline.org>,
	 Konrad Dybcio <konrad.dybcio@somainline.org>,
	 Rogerio Goncalves <rogerlz@gmail.com>,
	Martin Botka <martin@biqu3d.com>,
	 Martin Botka <martin.botka@somainline.org>
Subject: [PATCH 2/6] cpufreq: dt-platdev: Blocklist allwinner,h616 SoC
Date: Mon, 04 Sep 2023 17:57:02 +0200	[thread overview]
Message-ID: <20230904-cpufreq-h616-v1-2-b8842e525c43@somainline.org> (raw)
In-Reply-To: <20230904-cpufreq-h616-v1-0-b8842e525c43@somainline.org>

The AllWinner H616 uses H6 cpufreq driver.
Add it to blocklist so its not created twice

Signed-off-by: Martin Botka <martin.botka@somainline.org>
---
 drivers/cpufreq/cpufreq-dt-platdev.c | 1 +
 1 file changed, 1 insertion(+)

--- a/drivers/cpufreq/cpufreq-dt-platdev.c
+++ b/drivers/cpufreq/cpufreq-dt-platdev.c
@@ -102,6 +102,8 @@ static const struct of_device_id allowli
  */
 static const struct of_device_id blocklist[] __initconst = {
 	{ .compatible = "allwinner,sun50i-h6", },
+	{ .compatible = "allwinner,sun50i-h616", },
+	{ .compatible = "allwinner,sun50i-h618", },
 
 	{ .compatible = "arm,vexpress", },
 
